pytorch/inference/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -118,11 +118,14 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
  && rm ~/mambaforge.sh \
  && /opt/conda/bin/conda install -c conda-forge \
     python=$PYTHON_VERSION \
+    "wheel>=0.38.0" \
```

Comment:
This should be "0.38.1", since 0.38.0 has been yanked due to an unrelated non-security issue. (issue 470 on https://github.com/pypa/wheel)

Reply:
Sure, will update the version.